index.rst

```diff
@@ -176,18 +152,73 @@ If working with an individual CSC, which as an operator would be a rare occurren
     # Wait for script to execute
     await script.done()
 
-.. Or the LOVE interface.
+
+If these types of tasks are performed from the LOVE interface, then the same result occurs where the defaults are loaded automatically.
 
 .. TODO: Add example on how to launch script from LOVE interface
 
 
-.. _section-configuration-interation_non_default:
+Discovering Available Override Configurations
+---------------------------------------------
+
+The easiest way to get information from a CSC programmatically is by using a Jupyter notebook server.
+From a notebook, observers, developers and power users can easily interact with the system through Python.
+
+In order to know which configurations are available for a specific CSC, users can read the ``configurationsAvailable`` event.
+This is done using a ``salobj.Remote`` class.
+
+.. code-block:: python
+
+    from lsst.ts import salobj
+
+    domain = salobj.Domain()
+
+    atdome = salobj.Remote(domain, "ATDome")
+
+    await atdome.start_task()
+
+    config_available = await atdome.evt_configurationsAvailable.aget(timeout=5.)
+
+    # This will print the available filenames.
+    print(config_available.configurations)
+
+    # This will print the git hash of the loaded configuration repository
+    print(config_available.versions)
+
+Often, this can also be accomplished using a high-level class that is designed to interact with a group of CSCs.
```

Comment:
we should probably add this functionality to the high level class. It could work like:

```
config_available = await atcs.configurations_available()
```

and it returns a dictionary with the data for all components.

Or, if you are only interested in one component:

```
config_available = await atcs.configurations_available(components=["atdome"])
```

Then

```
    # This will print the available filenames.
     print(config_available["atdome"].configurations)
     # This will print the git hash of the loaded configuration repository
     print(config_available["atdome"].versions)
```

Reply:
I agree this would be useful. Create a ticket?

Reply:
yes

Reply:
https://jira.lsstcorp.org/browse/DM-29920